educationapplication/src/main/java/com/innowise/educationalsystem/exception/ApiError.java

```diff
@@ -0,0 +1,93 @@
+package com.innowise.educationalsystem.exception;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import org.hibernate.validator.internal.engine.path.PathImpl;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+
+import javax.validation.ConstraintViolation;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ApiError {
+    private HttpStatus status;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
+    private LocalDateTime timestamp;
+
+    private String message;
+
+    private String debugMessage;
```

Comment:
Id rather not send deails of the error to http client. It is better to generate unique error id, log the error with the id, and send the message with it